### DISCUSSION TITLE
MFA: "Remember Browser" Functionality

### DISCUSSION THREAD

**Discussion by appricosa** [ExpressionEngine/ExpressionEngine#3169] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
It can be quite frustrating, when you have to type in an MFA code with each and every login.
Virtually every other website has a "remember this browser" or "don't ask for a code on this browser again" checkbox.

**Describe the solution you'd like**
Kindly include an option to add current browser to trusted browsers/devices when asking for an MFA code.

**Describe alternatives you've considered**
Adding a checkbox to the MFA input form would be the only viable solution I suppose.

Thanks a lot.

**Comment by appricosa** [comment]
Hi, is this maybe on the list of a future update?

**Comment by MikeValentinePico** [comment]
This is an absolute must. Currently I am considering turning off MFA as it is more of a hindrance than a benefit.

Even if the device remembers you for 7 days or X amoutn of time it would be a huge step up. I had to enter my MFA code 9 times in one day as I needed EE access for 9 different updates throughout the day.
